server.js

```diff
@@ -17,17 +17,13 @@ app.get('/', function(req, res) {
   });
 });
 // or you can import route handlers from other files like this:
-const testRoutes = require('./routes/test');
-app.get('/test', testRoutes.getTestItems);
-app.post('/test', testRoutes.createTestItem);
-
-// TODO: delete the above dummy routes and add your actual routes
+const guitaristRoutes = require('./routes/guitarists');
+app.get('/guitarists', guitaristRoutes.getGuitarist);
+app.get('/:id', guitaristRoutes.getGuitaristById);
```

Comment:
also, I think you meant to make this endpoint `GET /guitarists/:id` rather than `GET /:id`